### PR TITLE
7.2.x - LRDOCS-7395 Update Setup Wizard documentation after LPS-100554 and LPS-84320 changes

### PR DESCRIPTION
**Pull request by jorgediaz-lr**
Hi @jhinkey 

Please, let me know in case I have to send this documentation update to other reviewer.

There are two LPS bugs that have modified Setup Wizard:
 * [LPS-100554](https://issues.liferay.com/browse/LPS-100554) - Added timezone drop-down, to be able to select default timezone of the system at setup time
 * [LPS-84320](https://issues.liferay.com/browse/LPS-84320) - Now, email field is empty by default, to discourage customers using default "test@liferay.com"

It is necessary to:
-  Update the screenshot as it is outdated
-  Explain "Default Timezone" configuration drop-down

Thank you

### CODE REVIEW

**Comment by jhinkey** [comment]
Thanks, @jorgediaz-lr ! I'll review them now. :-)

File: en/deployment/articles/01-deploying-liferay/04-installing-liferay.markdown [hunk 1]
```diff
@@ -81,6 +83,9 @@ For the administrator, supply the following information:
 
 **Email:** the default administrator user's email address
 
+| **Note:** the domain of administrator user's email will be used as the default
+| domain name of the instance (company.default.web.id configuration property).
```
Comment (jhinkey) [inline en/deployment/articles/01-deploying-liferay/04-installing-liferay.markdown]: @jorgediaz-lr  - By company.default.web.id configuration property, do you mean the [`company.default.web.id` *portal property*](http://docs.liferay.com/ce/portal/7.2-latest/propertiesdoc/portal.properties.html#Company)?

Comment (jorgediaz-lr) [reply en/deployment/articles/01-deploying-liferay/04-installing-liferay.markdown]: Hi @jhinkey , yes it is that `portal.properties` property

**Comment by jhinkey** [comment]
Sent https://github.com/sez11a/liferay-docs/pull/4661

**Comment by jhinkey** [comment]
@jorgediaz-lr - Thanks for updating the docs with these changes! :-)

**Pull request closed without merging**